manager/Makefile

```diff
@@ -94,6 +94,15 @@ undeploy_it:
 	cd config/manager && $(ABSTOOLBIN)/kustomize edit set image controller=${DOCKER_HOSTNAME}/${DOCKER_NAMESPACE}/manager:${DOCKER_TAGNAME}
 	$(TOOLBIN)/kustomize build --load_restrictor none config/integration-tests | $(TOOLBIN)/kubectl delete -f -
 
+deploy_cps: $(TOOLBIN)/kustomize $(TOOLBIN)/kubectl
```

Comment:
not a fan of shortcuts like `cps`, make has auto completion.

Reply:
OK